app/config/collections.php

```diff
@@ -1366,7 +1377,495 @@
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('default'),
+                'type' => Database::VAR_BOOLEAN,
+                'signed' => true,
+                'size' => 0,
+                'format' => '',
+                'filters' => [],
+                'required' => false,
+                'default' => false,
+                'array' => false,
+            ],
+            [
+                '$id' => ID::custom('credentials'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => ['json', 'encrypt'],
+            ]
+        ],
+        'indexes' => [
+            [
+                '$id' => ID::custom('_key_provider'),
+                'type' => Database::INDEX_KEY,
+                'attributes' => ['provider'],
+                'lengths' => [128],
+                'orders' => [Database::ORDER_ASC],
+            ],
+            [
+                '$id' => ID::custom('_key_name'),
+                'type' => Database::INDEX_FULLTEXT,
+                'attributes' => ['name'],
+                'lengths' => [128],
+                'orders' => [Database::ORDER_ASC],
+            ]
+        ],
+    ],
+
+    'messages' => [
+        '$collection' => ID::custom(DATABASE::METADATA),
+        '$id' => ID::custom('messages'),
+        'name' => 'Messages',
+        'attributes' => [
+            [
+                '$id' => ID::custom('providerId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('providerInternalId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('data'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16384,
+                'signed' => true,
+                'required' => true,
+                'default' => null,
+                'array' => false,
+                'filters' => ['json'],
+            ],
+            [
+                '$id' => ID::custom('to'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 16834,
```

Comment:
`16384` will limit each message to ~400 recipients, let's set this to the max of `65535` as well